contracts-periphery/src/interfaces/ISwapRouter.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.13;
+pragma abicoder v2;
```

Comment:
ABI Encoder V2 is the default as of 0.8.0 so this line can be removed

Reply:
Thanks! Removed them in https://github.com/ScopeLift/umbra-protocol/pull/341/commits/75faac7fcc1bbfed25b90ee04633145ddcd72693